frontend/src/components/forms/types/CreateFormValues.d.ts

```diff
@@ -0,0 +1,6 @@
+export type CreateFormValues = {
```

Comment:
Can we actually make this an interface and put it in the `interfaces` folder? `type` and `interface` are essentially the same thing in TS, but we're already using a ton of interfaces. Just a convention thing. Also don't need the `.d.ts`

Also though, probably don't need this at all once you add the Formik hook. If we type the form values using a generic, it should infer the types in the hook.